run_snafu.py

```diff
@@ -58,9 +58,16 @@ def main():
     #instantiate elasticsearch instance and check connection
     es={}
     if "es" in os.environ:
-        es['server'] = os.environ["es"]
-        es['port'] = os.environ["es_port"]
-        index_args.prefix = os.environ["es_index"]
+        if os.environ["es"] != ""
+            es['server'] = os.environ["es"]
+            logger.info("Using elasticsearch server with host:" + str(es['server']))
+        if os.environ["es_port"] != ""
+            es['port'] = os.environ["es_port"]
+            logger.info("Using elasticsearch server with port:" + str(es['port']))
```

Comment:
Can we also add a log message about the defaults when the es details are empty?

Reply:
We're not using any defaults with snafu, so if the user passes any vars as env then we'll show it otherwise there's no values.

Reply:
Yeah, something to the effect of `not indexing results for long term storage`